test/run.js

```diff
@@ -0,0 +1,20 @@
+const spawn = require('child_process').spawn
+
+const {start, stop} = require('./server')
+
+start(() => {
```

Comment:
I guess so, but it's also convenient to have this one line command for locally testing, right? And it doesn't cost much